src/main/java/com/regalii/regaliator/api/Response.java

```diff
@@ -61,6 +63,28 @@ public int httpCode() {
         return (Map<String, Object>) JSON.loadToMap(raw);
     }
 
+    public void printHeaders() {
```

Comment:
In either case just override the method `toString()` of `Object` and use it on your debugs.

Reply:
Just thought it might prove useful. I'm not using it in example code.